server.js

```diff
@@ -30,6 +30,16 @@ if ('development' === app.get('env')) {
   app.use(errorhandler());
 }
 
+var opn = require('opn');	//to open the url
```

Comment:
This package was been added in package.json
require it, to open the wavesexplorer, after runnint the server.